web/pdf_page_view.js

```diff
@@ -320,6 +331,7 @@ var PDFPageView = (function PDFPageViewClosure() {
     draw: function PDFPageView_draw() {
       if (this.renderingState !== RenderingStates.INITIAL) {
         console.error('Must be in new state before drawing');
+        return Promise.resolve(undefined);
```

Comment:
I see. Not sure if we want to change both. For pages, we fall through to start new rendering (e.g. for different scale). For thumbs, we don't care about scale or if image is set after page rendering. Maybe we just leave it without change, just add comments for intent? Or, we reject in both cases, it try to catch places when we are failing during testing?